cmd/acr/purge_test.go

```diff
@@ -606,6 +650,68 @@ var (
 	invalidLastUpdateTime = "date"
 
 	OneTagResult = &acr.RepositoryTagsType{
+		Response: autorest.Response{
+			Response: &http.Response{
+				StatusCode: 200,
+			},
+		},
+		Registry:  &testLoginURL,
+		ImageName: &testRepo,
+		TagsAttributes: &[]acr.TagAttributesBase{
+			{
+				Name:                 &tagName,
+				LastUpdateTime:       &lastUpdateTime,
+				ChangeableAttributes: &acr.ChangeableAttributes{DeleteEnabled: &deleteEnabled},
+				Digest:               &digest,
+			},
+		},
+	}
+
+	OneTagResultWithNext = &acr.RepositoryTagsType{
+		Response: autorest.Response{
+			Response: &http.Response{
+				StatusCode: 200,
+				Header:     http.Header{linkHeader: {"/acr/v1/&testRepo/_tags?last=latest&n=3&orderby=timedesc rel=\"next\""}},
+			},
+		},
+		Registry:  &testLoginURL,
+		ImageName: &testRepo,
+		TagsAttributes: &[]acr.TagAttributesBase{
+			{
+				Name:                 &tagName,
+				LastUpdateTime:       &lastUpdateTime,
+				ChangeableAttributes: &acr.ChangeableAttributes{DeleteEnabled: &deleteEnabled},
+				Digest:               &digest,
+			},
+		},
+	}
+
+	OneTagResultWithAmpersand = &acr.RepositoryTagsType{
+		Response: autorest.Response{
+			Response: &http.Response{
+				StatusCode: 200,
+				Header:     http.Header{linkHeader: {"/acr/v1/&testRepo/_tags?last=123%26latest&n=3&orderby=timedesc rel=\"next\""}},
+			},
+		},
+		Registry:  &testLoginURL,
+		ImageName: &testRepo,
+		TagsAttributes: &[]acr.TagAttributesBase{
+			{
+				Name:                 &tagName,
+				LastUpdateTime:       &lastUpdateTime,
+				ChangeableAttributes: &acr.ChangeableAttributes{DeleteEnabled: &deleteEnabled},
+				Digest:               &digest,
+			},
+		},
+	}
+
+	OneTagResultQueryEndingWithLast = &acr.RepositoryTagsType{
```

Comment:
What is the difference between `OneTagResultQueryEndingWithLast` and `OneTagResultWithAmpersand`? They look the same... Do you mean `</acr/v1/library/ubuntu/_tags?orderby=&n=100&last=123%26latest>; rel="next"` here?

Reply:
yes, i definitely meant that.